lib/code-callers.js

```diff
@@ -72,31 +108,32 @@ module.exports = {
         return callback(null);
       }
     };
+
     testFinished();
   },
 
-  getPythonCaller(callback) {
+  /**
+   * @param {string} coursePath
+   */
+  getPythonCaller(coursePath, callback) {
```

Comment:
I'm inclined to rename this to `getCodeCaller` - even though it's a caller _of_ Python code, it may be either a "python caller" or a "docker caller".

Alternatively, maybe I rename the code callers to `code-caller-native` and `code-caller-container` to mimic the values of `config.workersExecutionMode`? Naming is hard.